.tekton/ansible-chatbot-service-pull-request.yaml

```diff
@@ -266,7 +266,12 @@ spec:
           - name: COMMIT_SHA
             value: $(tasks.clone-repository.results.commit)
           - name: BUILD_ARGS
-            value: ["IMAGE_TAGS=pr-{{pull_request_number}} pr-{{pull_request_number}}.$(tasks.git-metadata.results.commit-timestamp)", "GIT_COMMIT=$(tasks.clone-repository.results.commit)", "RAG_CONTENTS_SUB_FOLDER=vector_db/aap_product_docs", "LIGHTSPEED_RAG_CONTENT_IMAGE=quay.io/ansible/aap-rag-content:latest", "LIGHTSPEED_RAG_EMBEDDINGS_IMAGE=quay.io/ansible/aap-rag-embeddings-image:latest"]
+            value:
+            - IMAGE_TAGS=pr-{{pull_request_number}} pr-{{pull_request_number}}.$(tasks.git-metadata.results.commit-timestamp)
+            - GIT_COMMIT=$(tasks.clone-repository.results.commit)
+            - RAG_CONTENTS_SUB_FOLDER=vector_db/aap_product_docs
+            - LIGHTSPEED_RAG_CONTENT_IMAGE=quay.io/ansible/aap-rag-content:1.0.1739753069
```

Comment:
This is for re-writing `BUILD_ARGS` in YAML list syntax and specifying a semver version to `aap-rag-content` image